pkg/controller/map/network/handler/openstack/handler.go

```diff
@@ -0,0 +1,116 @@
+package openstack
+
+import (
+	"path"
+	"strings"
+
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/web/ovirt"
+	"github.com/konveyor/forklift-controller/pkg/controller/watch/handler"
+	liberr "github.com/konveyor/forklift-controller/pkg/lib/error"
+	libweb "github.com/konveyor/forklift-controller/pkg/lib/inventory/web"
+	"github.com/konveyor/forklift-controller/pkg/lib/logging"
+	"golang.org/x/net/context"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+// Package logger.
+var log = logging.WithName("networkMap|ovirt")
+
+// Provider watch event handler.
+type Handler struct {
+	*handler.Handler
+}
+
+// Ensure watch on networks.
+func (r *Handler) Watch(watch *handler.WatchManager) (err error) {
+	w, err := watch.Ensure(
+		r.Provider(),
+		&ovirt.Network{},
+		r)
+	if err != nil {
+		return
+	}
+
+	log.Info(
+		"Inventory watch ensured.",
+		"provider",
+		path.Join(
+			r.Provider().Namespace,
+			r.Provider().Name),
+		"watch",
+		w.ID())
+
+	return
+}
+
+// Resource created.
+func (r *Handler) Created(e libweb.Event) {
+	if network, cast := e.Resource.(*ovirt.Network); cast {
```

Comment:
do we really set it to ovirt.Network or how is it that it doesn't fail?

Reply:
I suppose it isn't called yet 😅